app/styles/_color.scss

```diff
@@ -84,6 +84,7 @@ body {
   --font-weight-medium: 600;
   --font-weight-bold: 700;
 
+  --zindex-notification: 1099;
```

Comment:
why is this required ?

Reply:
It's from the previous commits. Once squashed, it should be fixed.